docs/configure-rest.md

```diff
@@ -343,6 +343,50 @@ the missing data won't be exported.
 
 See [Export Options](configure-rest.md#export_options)
 
+### Filter
+
+This guide provides instructions on how to use the `filter` feature in RestPerf. Filtering is useful when you need to query a subset of instances. For example, suppose you have a small number of high-value volumes from which you want Harvest to collect performance metrics every five seconds. Collecting data from all volumes at this frequency would be too resource-intensive. Therefore, filtering allows you to create/modify a template that includes only the high-value volumes.
+
+#### Objects (Excluding Workload)
+
+In RestPerf templates, you can set up filters under `counters`. Wildcards like `*` are useful if you don't want to specify all instances.
+
+For instance, to filter `volume` performance instances by volume name is `NS_svm_nvme` or contains `Test`, use the following configuration in RestPerf `volume.yaml` under `counters`:
```

Comment:
For instance, to filter `volume` performance instances by volume name, use the following configuration in RestPerf `volume.yaml` under `counters`. This example will return the volumes named `NS_svm_nvme` or the volumes with `Test` in their name.